Formula/anime-downloader.rb

```diff
@@ -3,8 +3,8 @@ class AnimeDownloader < Formula
 
   desc "Download your favourite anime"
   homepage "https://github.com/vn-ki/anime-downloader"
-  url "https://files.pythonhosted.org/packages/03/78/167cd1c2fe0ef43a0d137edc148279828ddd69b36b76e87f451b311b8a02/anime-downloader-5.0.1.tar.gz"
-  sha256 "d4e0da2f39edf5f6b4f9481b80a87fde951dbd3b379e8952a62bcc14b9d17371"
+  url "https://github.com/anime-dl/anime-downloader/archive/5.0.2.tar.gz"
```

Comment:
We should continue to use the `files.pythonhosted.org` URL if we can. This will allow resources to be bumped automatically